### ISSUE TITLE
ENOENT error on 

### ISSUE THREAD

**Issue by martinopic** [yoolab/iron-router-i18n#29]
Can't publish the package with npm dependency enabled if npm dependency is enabled in package.js I receive this error:

```
/Users/martino/.meteor/packages/meteor-tool/.1.0.40.959pzb++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/fibers/future.js:206
						throw(ex);
						      ^
Error: ENOENT, stat '/Users/martino/Sites/martino:iron-router-i18n/.npm/package/npm-shrinkwrap.json'
    at Object.Future.wait (/Users/martino/.meteor/packages/meteor-tool/.1.0.40.959pzb++os.osx.x86_64+web.browser+web.cordova/meteor-tool-os.osx.x86_64/dev_bundle/lib/node_modules/fibers/future.js:326:15)
```

**Comment by dohomi** [comment]
Hi @martinopic I checked out your package with the 0.5.x branch. not sure where you receive this error?

**Comment by martinopic** [comment]
Just a stupid error in my package.js, fixed with 0.5.1, sorry I needed to figure out this before going ahead with other issues.
